docs/sources/guides/whats-new-in-v5.md

```diff
@@ -135,7 +135,7 @@ We are introducing a new unique identifier (`uid`) in the dashboard JSON model.
 generated if not provided when creating a dashboard and will have a length of 9-12 characters.
 
 The unique identifier allows having persistent URLs for accessing dashboards, sharing them
-between instances and when using [dashboard provisioning](#dashboards). This means that dashboard can
+between instances and when using [dashboard provisioning]((/administration/provisioning/#reusable-dashboard-urls)). This means that dashboard can
```

Comment:
The link isn't broken exactly - it's referring to the section above but that doesn't seem to work too well in practice when you click on the link.

If linking to the dashboard provisioning docs, shouldn't the url be: `/administration/provisioning/#dashboards`

Reply:
I linked to the section that matched the content of the change.